ui/utils.py

```diff
@@ -170,3 +178,126 @@ def write_x509_files():
     """Write the x509 certificate and key to files"""
     write_to_file(settings.MIT_WS_CERTIFICATE_FILE, settings.MIT_WS_CERTIFICATE)
     write_to_file(settings.MIT_WS_PRIVATE_KEY_FILE, settings.MIT_WS_PRIVATE_KEY)
+
+
+def get_video_analytics(video_key):
+    """Get video analytics data from Google Analytics."""
+    ga_client = get_google_analytics_client()
+    ga_response = ga_client.reports().batchGet(
+        body=generate_google_analytics_query(video_key)).execute()
+    try:
+        return parse_google_analytics_response(ga_response)
+    except Exception as exc:
+        raise GoogleAnalyticsException(
+            'Could not parse analytics response') from exc
+
+
+def get_google_analytics_client():
+    """Gets a Google Analytics client.
+
+    Returns:
+        analytics_client: An analytics client
+    """
+    try:
+        if not settings.GA_KEYFILE_JSON:
+            raise RuntimeError("missing 'settings.GA_KEYFILE_JSON'")
```

Comment:
This works fine, but for consistency it might be better to remove this and add `GA_VIEW_ID` and `GA_KEYFILE_JSON` to [settings.MANDATORY_SETTINGS](https://github.com/mitodl/odl-video-service/blob/624ef5e5bd86783a312b7deb8a359b7f1df7e6c7/odl_video/settings.py#L414) instead.

Reply:
Forgot that for anything in `MANDATORY_SETTINGS`, a dummy value has to be placed in the `tox.ini` file, under the `setenv=` section.  Once you add these the travis tests should pass.

Reply:
GA_DIMENSION_CAMERA, GA_KEYFILE_JSON, GA_VIEW_ID